eng/pipelines/templates/stages/archetype-cpp-release.yml

```diff
@@ -5,7 +5,7 @@ parameters:
 stages:
   - ${{if and(eq(variables['Build.Reason'], 'Manual'), eq(variables['System.TeamProject'], 'internal'))}}:
     - ${{ each artifact in parameters.Artifacts }}:
-      - stage: Release_${{artifact.safeName}}
+      - stage: Release_${{ replace(artifact.Name, '-', '_') }}
```

Comment:
nit: lowercase n?

Reply:
@mitchdenny is this a potential fix for needing the safeName in other languages?

Reply:
Yes .... I was thinking the same thing.